bsp/Src/UART.c

```diff
@@ -380,7 +388,6 @@ void HAL_UART_TxCpltCallback(UART_HandleTypeDef *huart) {
         return;
     }
```

Comment:
This (see lines 369-381) fails compilation because of an "else without an if" for processors that don't have UART4 or UART5 defined. I suppose the cleanest way to do this is to make an ifdef for the else portion that's just 
`#if !defined(UART4) && !defined(UART5)`
around the return, but lowk this kinda ugly. lmk if you think of something better.

Reply:
Actually, the callback will never trigger if neither UART is defined, so you can just remove the else.